src/include/type/value_factory.h

```diff
@@ -63,9 +63,9 @@ class ValueFactory {
   }
 
   static inline Value GetVarcharValue(
-      const char *value, UNUSED_ATTRIBUTE VarlenPool *pool = nullptr) {
+      const char *value, bool manage_data, UNUSED_ATTRIBUTE VarlenPool *pool = nullptr) {

```

Comment:
You should add a function `@param` comment that explains what `manage_data` means in case anybody calls it. Do this for all of the Get*Value methods.